pkg/integrations/cmdrequest/handler.go

```diff
@@ -12,9 +12,10 @@ import (
 )
 
 var (
+	// helpers for testing purposes
 	NoopHandleFn    = func(protocol.CmdRequestV1) {}
 	ErrNotSupported = errors.New("integration instance lookup not supported for cmd request handler")
-	noLookup        = integration.InstancesLookup{
+	NoLookup        = integration.InstancesLookup{
```

Comment:
I cant find any usage of this `NoLookup`, is it needed?

Reply:
nope, I left that for testing purposes, but I can delete it as well